scripts/purge_queue.py

```diff
@@ -0,0 +1,109 @@
+import sys
+
+import argparse
+import boto3
+import json
+import more_itertools
+from azul import config
+import logging
+
+from urllib.parse import urlparse
+
+logger = logging.getLogger(__name__)
+parser = argparse.ArgumentParser(description=__doc__)
```

Comment:
For this to work you need to actually provide a docstring at module scope.

Reply:
I'd prefer not to define the parser at module scope. Put all of this and the body of the `if __name__ …` into a main() function that has the following signature: `def main(args:List[str]) -> None`. Call that function in `if __name__ …` like this `main(sys.argv[1:])`. I'll explain in person.